Lib/test/test_ast/test_ast.py

```diff
@@ -3090,16 +3090,16 @@ def assert_ast(self, code, non_optimized_target, optimized_target):
             f"{ast.dump(optimized_tree)}",
         )
 
+    def create_binop(self, operand, left=ast.Constant(1), right=ast.Constant(1)):
```

Comment:
```suggestion
    def create_binop(self, operand, left=None, right=None):
```
please, don't use complex objects as defaults

Reply:
This seems fine as is; it's more convenient to write many of the remaining tests this way.

Reply:
I am in a purist camp :) But, this is minor, I agree. Since these objects aren't ever mutated, this does not introduce any problems.

Reply:
Yeah, I also don't like mutable defaults. But this one looks not very complex and it doesn't seem to be mutated anytime soon :) We can rewrite it, but the diff would be much larger